tests/e2e/framework/rawVM.go

```diff
@@ -42,7 +42,7 @@ var (
 	clusterName   = flag.String("cluster_name", "", "The name of the istio cluster that the VM extends")
 	image         = flag.String("image", "debian-9-stretch-v20170816", "Image Name")
 	imageProject  = flag.String("image_project", "debian-cloud", "Image Project")
-	proxyTag      = flag.String("proxy_tag", "", "proxy tag that identifies a debian pkg")
+	sidecarTag    = flag.String("sidecar_tag", "", "proxy tag that identifies a debian pkg")
```

Comment:
Can you rename it to something like `sidecar_debian_tag` or something, just to make it clear.

Reply:
Done